Source/Shaders/Builtin/Functions/octDecode.glsl

```diff
@@ -1,3 +1,29 @@
+ /**
+  * Decodes a unit-length vector in 'oct' encoding to a normalized 3-component Cartesian vector.
+  * The 'oct' encoding is described in "A Survey of Efficient Representations of Independent Unit Vectors",
+  * Cigolle et al 2014: http://jcgt.org/published/0003/02/01/
+  *
+  * @name czm_octDecode
+  * @param {vec2} encoded The oct-encoded, unit-length vector
+  * @param {float} range The maximum value of the SNORM range. The encoded vector is stored in log2(rangeMax+1) bits.
+  * @returns {vec3} The decoded and normalized vector
+  */
+  vec3 czm_octDecode(vec2 encoded, float range)
```

Comment:
I'll be pleasantly surprised if JSDoc handles function overloads, but it's not an issue now.